stellarphot/io/tess.py

```diff
@@ -474,7 +474,13 @@ def _retrieve_target_file(self):
         result = requests.get(
             self.aperture_server + "cgi-bin/gaia_to_aij/upload_request.cgi",
             params=params,
+            timeout=15,  # If no response in 15 seconds we won't ever get one...
         )
+        if result.status_code != 200:
```

Comment:
I don't think this has to do with name field restrictions. :)

Reply:
Yeah, we were getting a test failure on two tests because of this